float32 temperature
float32 mcu_voltage
float32 drive_voltage
float32 mcu_current
float32 drive_current
float32[] rpm